src/schemas.ts

```diff
@@ -800,6 +800,48 @@ const foxglove_LaserScan: FoxgloveMessageSchema = {
   ],
 };
 
+const foxglove_EventKeyValue: FoxgloveMessageSchema = {
+  type: "message",
+  name: "EventKeyValue",
+  description: "a single key-value pair of metadata for an event",
+  fields: [
+    {
+      name: "key",
+      type: { type: "primitive", name: "string" },
+      description: "key for this metadata",
+    },
+    {
+      name: "value",
+      type: { type: "primitive", name: "string" },
+      description: "value of the metadata",
+    },
+  ],
+};
+
+const foxglove_Event: FoxgloveMessageSchema = {
+  type: "message",
+  name: "Event",
+  description: "a message used to annotate some event of interest in the data",
```

Comment:
nit: I believe we are capitalizing sentences for descriptions on the other types/fields in this file